makeflow/src/makeflow.c

```diff
@@ -2119,6 +2152,14 @@ int main(int argc, char *argv[])
 			case LONG_OPT_TLQ:
 				tlq_port = atoi(optarg);
 				break;
+			case LONG_OPT_FILE_STATUS:
+				file_status_on = 1;
+				file_status_name = optarg;	
+				break;
+			case LONG_OPT_FILE_STATUS_INTERVAL:
+				file_status_on = 1;
+				file_status_interval = atoi(optarg);
```

Comment:
You can use the function `string_parse_metric` in `dttools/src/stringtools.h` which parses various formats of hours, minutes, seconds, etc.